app/controllers/batch_connect/session_contexts_controller.rb

```diff
@@ -22,16 +15,12 @@ def new
         #{@app.validation_reason} Please contact support if you see this message
       EOT
     end
-  rescue BatchConnect::App::AppNotFound => e
```

Comment:
So I'll just remove "Please contact support if you see this message". If you object lets discuss offline.